intellij-plugin-structure/structure-base/src/main/kotlin/com/jetbrains/plugin/structure/base/problems/PluginProblems.kt

```diff
@@ -23,3 +29,36 @@ val PluginProblem.isInvalidDescriptorProblem: Boolean
   } else {
     this is InvalidDescriptorProblem
   }
+
+fun PluginProblem.isInstance(pluginProblemClass: KClass<*>): Boolean =
+  pluginProblemClass.isInstance(unwrapped)
+
+
+private const val PLUGIN_PROBLEM_PACKAGE_DEFAULT_PREFIX = "com.jetbrains.plugin.structure."
+
+object PluginProblems {
+  /**
+   * Resolves the problem ID to a fully qualified Kotlin class.
```

Comment:
What is "a fully qualified Kotlin class"? Is it just Kotlin class? If so, then please change to:
> Resolves a Kotlin class by a given problem ID.

Reply:
Resolved.